src/profile/utils.js

```diff
@@ -44,15 +44,29 @@ function getDataItem(data, itemName) {
   }
 }
 
-function displayList(data, userInfoList) {
-  for (const listItem in data) {
-    const innerText = `${formatPropertyField(listItem)}: ${getDataItem(
-      data,
-      listItem,
-    )}`;
+function displayList(primaryData, userInfoList, secondaryData) {
+  const prevSibling = userInfoList.previousSibling.innerHTML;
+  const isOldData = prevSibling.toLowerCase().includes('old');
+  const isNewData = prevSibling.toLowerCase().includes('new');
+
+  for (const listItem in primaryData) {
+    const oldValue = getDataItem(primaryData, listItem);
+    const newValue = getDataItem(secondaryData, listItem);
+    const isValueEqual = String(oldValue).trim() === String(newValue).trim();
+
+    let diffClass;
+    if (!isValueEqual) {
+      diffClass = isOldData ? 'oldDiff' : 'newDiff';
```

Comment:
we can create constants for 'oldDiff' and 'newDiff' strings.

Reply:
Can you please check now?